client/src/pages/formatFile.js

```diff
@@ -1,10 +1,15 @@
 import React, { useEffect, useState } from "react";
 import { useLocation, useParams } from "react-router-dom";
-import History from "../components/history.js";
 
 const formatFile = () => {
   const location = useLocation();
+
   const [data, setData] = useState(null);
+  const [title, setTitle] = useState("");
+  const [remix, setRemix] = useState("Remix");
```

Comment:
this is unused now, make it hardcoded remix in the button

Reply:
Resolved, ty